core/utils.go

```diff
@@ -75,6 +75,13 @@ func isPrimary(view uint64, id uint32, n uint32) bool {
 	return uint64(id) == view%uint64(n)
 }
 
+func getTrimmedMsg(msg string) string {
```

Comment:
How about naming this function just "trim" or "trimMessage"? I imagine retrieving a value from a member variable of a object from the word "get".

Reply:
Thanks, I'll do it.

Reply:
One more option for naming: `shortString`.